releases.json

```
@@ -1227,6 +1227,14 @@
       "1.2.158-1"
     ]
   },
+  "wayland-protocols": {
+    "dependency_names": [
+      "wayland-protocols"
+    ],
+    "versions": [
+        "1.24"
```

Needs to have a wrap revision too, not just an upstream release version.

```suggestion
        "1.24-1"
```